src/components/LoginSignUp/Login.js

```diff
@@ -171,13 +172,12 @@ const Login = ( ): Node => {
         <Text style={textStyles.forgotPassword}>{t( "Forgot-Password" )}</Text>
       </TouchableOpacity>
       {error && <Text style={textStyles.error}>{error}</Text>}
-      <RoundGreenButton
-        style={viewStyles.button}
-        buttonText="Log-in"
-        handlePress={login}
+      <Button
+        level="primary"
+        text="Log-in"
+        onPress={login}
         disabled={!email || !password}
         testID="Login.loginButton"
-        loading={loading}
```

Comment:
I removed the `loading` prop since loading was passed as a prop for `Button` element from `react-native-paper`. Let me know if we still want to keep it.

Reply:
We definitely want to keep it, otherwise the Login view cannot show that it is in a loading state. I guess I forgot to include this in the spec, so apologies for that, but the new `<Button>` component should have a `loading` prop that shows an activity indicator when true.

One simple way to do this would be to use the `react-native-paper` `<Button>` component inside our own `<Button>` component, so ours is basically a wrapper that accepts the props we want and configures the `react-native-paper` one appropriately.